tools/serverpod_cli/lib/src/database/extensions.dart

```diff
@@ -355,16 +355,37 @@ extension ForeignKeyDefinitionPgSqlGeneration on ForeignKeyDefinition {
 
     var refColumsFmt = referenceColumns.map((e) => '"$e"');
 
+    var delete = onDelete != null ? onDelete?.toPgSqlAction() : 'CASCADE';
+    var update = onUpdate != null ? onUpdate?.toPgSqlAction() : 'NO ACTION';
```

Comment:
Suggestion: Make onDelete and onUpdate not nullable and use their default value.

Don't we always configure onDelete and onUpdate with default values already?

Reply:
Indeed, I tried that at first, perhaps we should still do that. My problem was that this is set in generated code, this generated code is used in a lot of places and I didn't want to accidentally break anything. I think we can take a stab at this in another PR, and also make sure we have enough test coverage for the code we touch.